python/pypto/ir/roundtrip.py

```
@@ -0,0 +1,102 @@
+    _after_modes = (_passes.VerificationMode.AFTER, _passes.VerificationMode.BEFORE_AND_AFTER)
+    before_cb = _before if mode in _before_modes else None
+    after_cb = _after if mode in _after_modes else None
+
+    return _passes.CallbackInstrument(
+        before_pass=before_cb,
+        after_pass=after_cb,
+        name="RoundtripInstrument",
+    )
+
+
+__all__ = ["verify_roundtrip", "RoundtripInstrument"]
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)